src/yunohost/domain.py

```diff
@@ -273,7 +273,7 @@ def domain_dns_conf(domain, ttl=None):
             '(?=.*(;[\s]*|")p=(?P<p>[^";]+))'), dkim_content, re.M|re.S
         )
         if dkim:
-            result += '\n{host}. {ttl} IN TXT "v={v}; k={k}; p={p}"'.format(
+            result += '\n{host} {ttl} IN TXT "v={v}; k={k}; p={p}"'.format(
```

Comment:
Are you sure?

Tbh we've tried to do that on the domain name of a member and gandi rejected the file telling that the syntax was invalid, I didn't had the time to dig this. I'm at the work but I'll try to remember to check the domain for which we did that but from what I understand we'll probably need to handle both full domain and subdomain case no?